recipes-kernel/linux/files/5.x-kernel/0007-Add-support-for-U9300C-TD-LTE-module.patch

```diff
@@ -1,7 +1,7 @@
-From 0119ef2888c7cc3c9edd9525bfea2411594eea5c Mon Sep 17 00:00:00 2001
+From 657c8bf11d8da1698ccbc16accf3af8cb859b67c Mon Sep 17 00:00:00 2001
 From: Su Bao Cheng <baocheng.su@siemens.com>
 Date: Tue, 23 Apr 2019 10:07:17 +0800
-Subject: [PATCH 02/14] Add support for U9300C TD-LTE module
+Subject: [PATCH 07/14] Add support for U9300C TD-LTE module
```

Comment:
I think we are missing a follow-up (v2, fixed) on https://lkml.org/lkml/2020/6/16/346. Please do that now, it change is trivial enough.

Reply:
yeah,I would check the status with related colleagues